ftest/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
```

Comment:
This now spins up 3 different instances of shaas with different config. Tests can now use the `TestingEnvironmentService` to specify which service is being targeted.